tests/integration/generate_fixtures/go_ethereum.py

```diff
@@ -361,6 +361,50 @@ def setup_chain_state(w3):
     return geth_fixture
 
 
+def update_doc_version(new_version):
+    file_path = "./docs/contributing.rst"
+    with open(file_path) as file:
+        lines = file.readlines()
+
+    replaced = False
+    for i, line in enumerate(lines):
+        if "python -m geth.install " in line:
+            lines[i] = f"      $ python -m geth.install {new_version}\n"
+        if "GETH_BINARY=~" in line:
+            lines[
+                i
+            ] = f"      $ GETH_BINARY=~/.py-geth/geth-{new_version}/bin/geth python ./tests/integration/generate_fixtures/go_ethereum.py\n"  # noqa: E501
+            replaced = True
+            break
+
+    if not replaced:
+        raise ValueError("`geth_version` for conftest not found / replaced.")
+
+    with open(file_path, "w") as file:
+        file.writelines(lines)
+    print(f"Updated geth_version in docs to {new_version}")
+
+
+def update_fixture_generation_version(new_version):
+    file_path = "./tests/integration/go_ethereum/conftest.py"
```

Comment:
We could add the benchmarking path here as well `./web3/tools/benchmark/node.py`